### DISCUSSION TITLE
Is tsx supported with inline code blocks?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [mkdocs/mkdocs#3458] [Q&A] [answered]
I'm writing docs using mkdocs with Backstage and `tsx` doesn't highlight the code. `typescript` works, but not `tsx`.

```
 ```tsx
 const NewPage = () => {
   return <Typography variant="h1">Heading</Typography>
 };
 ```

**Comment by oprypin** [comment, marked as answer]
You didn't specify the theme.
If it's one that uses pygments, you can follow this issue
* https://github.com/pygments/pygments/issues/1815

**Reply by cmacdonnacha** [reply]
Great thanks.
